deb-package/build

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Apache License, Version 2.0.
+# (See accompanying LICENSE file or copy at
+# https://www.apache.org/licenses/LICENSE-2.0.txt)
+set -xeu
+
+# Set up package
+pkgname=libmxnet
+pkgversion_number=$(git describe --tags --dirty |
+                    sed 's/^v\(.*\)$/\1/' |
+					sed 's/+socio/-socio/' |
+                    sed 's/-\([0-9]\+\)-\(g[0-9a-f]\+\)/+\1~\2/' |
+                    sed 's/\(~g[0-9a-f]\+\)-dirty$/-dirty\1/' |
+                    sed 's/-dirty/~dirty.'`date +%Y%m%d%H%M%S`'/'
+                   )
+pkgdistro=$(lsb_release -cs)
+pkgversion="${pkgversion_number}~${pkgdistro}"
+pkgmaintainer="Sociomantic Tsunami <tsunami@sociomantic.com>"
+
+genchangelog()
+{
+    echo "$1 ($2) $(lsb_release -sc); urgency=low"
+    echo
+    prevtag=$(git describe --tags --abbrev=0 HEAD^)
+    git log --date=short --format=" * %s (%h, $cd)" "$prevtag"..HEAD |
```

Comment:
This is generating an error in the build log:

```
./deb-package/build: 29: ./deb-package/build: cd: parameter not set
```

It looks like a typo: it should probably be `%cd`.